cio/io.go

```diff
@@ -241,6 +241,20 @@ func LogURI(uri *url.URL) Creator {
 	}
 }
 
+// LogURI provides the raw logging URI
+// It also sets the terminal option to true
```

Comment:
```suggestion
// TerminalLogURI provides the raw logging URI
// as well as sets the terminal option to true.
```

Reply:
Thanks @dcantah , It has been changed :-)